src/relay/backend/vm/compiler.cc

```diff
@@ -328,6 +328,7 @@ class VMFunctionCompiler : DeviceAwareExprFunctor<void(const Expr& n)> {
       case Opcode::Ret:
       case Opcode::Goto:
       case Opcode::Fatal:
+      case Opcode::KillRegister:
```

Comment:
You probably need to update the VM byte code serialization code as this will break if you serialize a model containing the instruction.

Reply:
I didn't see serialize.cc at first brush so I thought it might be missing.